tools/cdk-release/lib/private/print.ts

```diff
@@ -0,0 +1,23 @@
+import * as util from 'util';
+import { ReleaseOptions } from '../types';
+
+export function debug(opts: ReleaseOptions, message: string): void {
+  if (opts.verbose) {
+    // eslint-disable-next-line no-console
+    console.log(`[cdk-release] ${message}`);
+  }
+}
+
+export function debugObject(opts: ReleaseOptions, message: string, object: any): void {
+  if (opts.verbose) {
+    // eslint-disable-next-line no-console
+    console.log(`[cdk-release] ${message}:\n${JSON.stringify(object, null, 2)}`);
```

Comment:
Why not `util.inspect` or have `console.log` format this object?

Reply:
I don't know, it's a debug function, I really haven't spent much time polishing it 🙂.

Let me know what exactly you'd rather see here.

Reply:
Let me put it like this:

💅 How about:

```ts
console.log(`[cdk-release] ${message}:\n`, object); 
```

Reply:
Sure. Changed.